finetuner/constants.py

```diff
@@ -19,3 +19,4 @@
 EVAL_DATA = 'eval_data'
 FINETUNED_MODELS_DIR = 'finetuned_models'
 MODEL = 'model'
+ARTIFACT_IDS = 'FINETUNER_ARTIFACT_IDS'
```

Comment:
as mentioned above, it is the env variable we send to AWS batch, not the field we saved in the database.

Reply:
but how do I, as a client, retrieve it then?